README.md

```diff
@@ -92,7 +85,7 @@ located in the `astrobee_android/` folder.
 If you are a NASA user, please make sure the `android` submodule was checked out
```

Comment:
The build is totally separate... kind of thinking we should just delete android submodule. Thoughts?

Reply:
Yeah I agree, we should still reference it in the main page such that users are aware of it, also add mention of the GDS repo in the main readme.
I'll look into how to make submodules optional. Maybe we can add it to the install instructions, clearly stating what the repos are for and letting the user decide according to the needs as an add-on?